contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -588,20 +650,15 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
       // DON config count starts at index 1
       if (don.configCount == 0) revert DONDoesNotExist(donId);
       delete s_dons[donId];
+      s_removedDONIds.add(donId);
       emit ConfigSet(donId, 0);
     }
   }
 
   /// @notice Gets DON's data
   /// @param donId The DON ID
-  /// @return uint32 The DON ID
-  /// @return uint32 The DON's config count
-  /// @return bool True if the DON is public
-  /// @return bytes32[] The list of node P2P IDs that are in the DON
-  /// @return CapabilityConfiguration[] The list of capability configurations supported by the DON
-  function getDON(
-    uint32 donId
-  ) external view returns (uint32, uint32, bool, bytes32[] memory, CapabilityConfiguration[] memory) {
+  /// @return DONParams The DON's parameters
+  function getDON(uint32 donId) public view returns (DONParams memory) {
```

Comment:
Could we make an internal function for the getter as opposed to making this a public function? The goal would be to clearly communicate that this is meant only for off-chain use.